src/components/enketo/preview.vue

```diff
@@ -35,7 +35,8 @@ export default {
     formVersion: {
       type: Object,
       required: true
-    }
+    },
+    outlined: Boolean
```

Comment:
I wish there was a easier/simpler way to achieve the desire result with css-class as pass-through attribute instead of having a prop here